api/src/main/java/org/pmiops/workbench/cdr/model/DbCriteria.java

```diff
@@ -109,17 +111,30 @@ public void setSelectable(boolean selectable) {
   }
 
   @Column(name = "est_count")
-  public String getCount() {
-    return count;
+  public Long getCount() {
+    return this.count == null ? 0 : this.count;
   }
 
-  public void setCount(String count) {
+  public void setCount(Long count) {
     this.count = count;
   }
 
-  @Transient
-  public Long getLongCount() {
-    return Strings.isNullOrEmpty(this.count) ? null : Long.valueOf(this.count);
+  @Column(name = "rollup_count")
```

Comment:
You need to add parentCount and childCount to the `equals()` and `hashCode()` methods

Reply:
Done